tests/sklearn/test_sklearn_models.py

```diff
@@ -20,10 +20,6 @@
   - calls to predict_proba
   - calls to decision_function
 
-Are currently missing
-  - check of predict_proba
-  - check of decision_function
-
 More information in https://github.com/zama-ai/concrete-ml-internal/issues/2682
```

Comment:
is this comment linked to what you deleted above ?

Reply:
seems like an old issue indeed, I'll remove it in this PR, thanks !